_pages/how-we-work/how-to-log-in.md

```diff
@@ -51,8 +52,12 @@ VMware is utilized to access your GSA Microsoft Windows desktop.  Some governmen
 
 AnyConnect is utilized as a GSA VPN client so that you can use a variety of systems that require a VPN directly from your local computer. Go [here]({{site.baseurl}}/anyconnect) for more information on configuring this tool.
 
+## Network connections
+To learn how to connect to the GSA network on campus and remotely, read the [GSA Network Connection Guide](https://docs.google.com/document/d/1nBNXt6Ov4KWmpz6y9rgKw93mxZucVsoYC4PFABTeIA4/edit#heading=h.6hzaqzho5cc3)
```

Comment:
Want to break this change out to its own pull request?

Reply:
I think that's a great idea. https://github.com/18F/handbook/pull/1624